diff --git a/node_modules/@react-navigation/bottom-tabs/src/views/BottomTabBar.tsx b/node_modules/@react-navigation/bottom-tabs/src/views/BottomTabBar.tsx
index 96469a2..f8cd968 100644
--- a/node_modules/@react-navigation/bottom-tabs/src/views/BottomTabBar.tsx
+++ b/node_modules/@react-navigation/bottom-tabs/src/views/BottomTabBar.tsx
@@ -284,11 +284,11 @@ export default function BottomTabBar({
           paddingHorizontal: Math.max(insets.left, insets.right),
         },
         tabBarStyle,
+        { pointerEvents: isTabBarHidden ? 'none' : 'auto' },
       ]}
-      pointerEvents={isTabBarHidden ? 'none' : 'auto'}
       onLayout={handleLayout}
     >
-      <View pointerEvents="none" style={StyleSheet.absoluteFill}>
+      <View style={[StyleSheet.absoluteFill, { pointerEvents: 'none' }]}>
         {tabBarBackgroundElement}
       </View>
       <View accessibilityRole="tablist" style={styles.content}>
